### ISSUE TITLE
Question: Direct Query / Dataflows

### ISSUE THREAD

**Issue by schuffa** [otykier/TabularEditor#514]
Hello,

I have connected to a data model using direct query for dataflows. I made a couple of enhancements (calculation groups, perspectives, etc.) and deployed - all fine! Now I want to add a further dataflow (again DQ). How can I do that from Tabluar Editor end? Can you help me navigating to the right spot in the documentation or give me a brief advice here?

Many thanks already and best regards,

Andreas

**Comment by otykier** [comment]
I haven't had the opportunity to use Tabular Editor with dataflows yet, so perhaps someone with more experience than me can chime in, but I think a good place to start would be to look at the other tables in the model. Maybe you can copy an existing table, modify the partition M expression and add/delete columns based on what that dataflow contains.

Also, even though Tabular Editor can't parse an M expression and infer data columns automatically from a dataflow, you may still be able to automate some of the column mapping, by using the [approach outlined here](https://github.com/otykier/TabularEditor/wiki/Importing-Tables#manually-importing-schemametadata).

Please feel free to report back with any findings, or if you hit any issues along the way! Thanks!

**Comment by schuffa** [comment]
thanks for the quck reply. It seems that there is an issue with this functionality, as even deployments frmo PB Desktop to PBI Service do currently cause an error. I treid the copy & paste route in Tabluar Editor, but if the feature as such provides currently errors at least in my scenario) then those needs to be fixed first. I recognized that only after my post , meanwhile I have a support ticket with MS and a meeting scheduled for tomorrow. As soon as this issue is fixed, I will see if something remains on Tabluar Editor end. Will keep you posted! Thanks again for the quick reply!

**Comment by otykier** [comment]
Closing this issue for now. Feel free to reopen or post a new issue if it turns out there's a bug in Tabular Editor regarding dataflows.
